packages/copyable/src/Copyable.tsx

```diff
@@ -178,6 +178,7 @@ export default function Copyable({
         justify={Justify.Middle}
         trigger={trigger}
         triggerEvent={TriggerEvent.Click}
+        usePortal={false}
```

Comment:
Are there any cases where the copyable tooltip _should_ use a portal?

Reply:
We might want to expose a prop here:

shouldPortalTooltip comes to mind but open to others, we can default to `false` as you have here.

Reply:
Actually, let's default to `true`, in which case this is only a minor change rather than a breaking one.

Reply:
(that's also more in line with how we default components that consume our Portal component)